Common/MVKCommonEnvironment.h

```diff
@@ -85,7 +88,17 @@ extern "C" {
 
 /** Building for macOS, iOS or visionOS. */
 #ifndef MVK_MACOS_OR_IOS_OR_VISIONOS
-#	define MVK_MACOS_OR_IOS_OR_VISIONOS	(MVK_MACOS || MVK_IOS | MVK_VISIONOS)
+#	define MVK_MACOS_OR_IOS_OR_VISIONOS	(MVK_MACOS || MVK_IOS || MVK_VISIONOS)
```

Comment:
I know you didn't originally add this, but could this be removed, and `!MVK_TVOS` used instead?

Reply:
would make sense, I did find this was getting out of hand. I suppose new platform don't get added very often but seems like a brittle setup (not that I can think of anything better right now)